src/components/booking.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable consistent-return */
+/* eslint-disable array-callback-return */
+import React, { useState } from 'react';
+import { useGetBookingsQuery } from '../services/hotel';
+import Loader from './Loader';
+
+export default function Booking() {
```

Comment:
- [ ] Good job adding a search bar!  However, in the "Bookings" section, the moment I try to search for anything I get a white screen.  Kindly fix this bug so your app works as intended. 💪